roles/arteria-delivery-ws/templates/delivery_app.config.j2

```diff
@@ -12,3 +12,4 @@ path_to_mover: "{{ mover_path }}"
 project_links_directory: "{{ delivery_links }}"
 dds_conf:
   log_path: "{{ arteria_service_log_dir }}/dds.log"
+  mount_dir: "{{ lookup('env', 'SNIC_TMP') }}"$
```

Comment:
one thing to bear in mind is that on the login nodes, the scratch partition is shared between all users on the system. this is not suitable for sensitive personal data (which many deliverables will be). On the compute node, the designated scratch area should be private to the user but I'm not entirely convinced that the `SNIC_TMP` variable gets correctly set.. you may need to confirm that.

Reply:
Hmm good point, I guess the delivery service will only run from the login node, right? What would be a good place for the DDS staging directories then? Would `/proj/ngi2016001/nobackup/private/workspace_funk_004` be acceptable? I guess we need to make sure we clean these directories once data has been delivered as well (one advantage of `/scratch` was that it would automatically erase old data when space is needed).

Reply:
Oh, yeah actually that should work, I just need to make sure these are cleaned after the delivery. I'll fix a new version of `arteria-delivery` and update this PR, thanks! :+1: